tests/MSBuildDeviceIntegration/Tests/XASdkDeployTests.cs

```diff
@@ -156,6 +157,70 @@ public void DotNetDebug ()
 			WaitFor (2000);
 			Assert.IsTrue (breakpointHit, "Should have a breakpoint");
 		}
+
+		[Test]
+		public void TypeAndMemberRemapping ([Values (false, true)] bool isRelease)
```

Comment:
@dellis1972 , @jonathanpeppers : please review this test for sanity. :-)

(I have no idea what I'm doing here. [dog-on-keyboard.gif])